support-frontend/assets/pages/contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -55,6 +59,8 @@ const mapDispatchToProps = (dispatch: Function) => ({
 
 function withProps(props: PropTypes) {
   const contributionTypes = props.contributionTypes[props.countryGroupId];
+  const isLabelTestVariant = props.landingPageChoiceArchitectureLabelsTestVariants === 'withLabels';
+  const createContributionTypeLabel = isLabelTestVariant ?  toHumanReadableContributionTypeAbTest : toHumanReadableContributionType;
```

Comment:
the build failed because of this line: `Line 63 exceeds the maximum line length of 120  max-len`

Reply:
i'm usually so good about linting!